playwright.config.ts

```diff
@@ -19,7 +19,7 @@ const config: PlaywrightTestConfig = {
   use: {
     baseURL: `${process.env.E2E_BASE_URL}/spa/`,
     storageState: 'e2e/storageState.json',
-    video: 'retain-on-failure',
+    video: 'on',
```

Comment:
Did we mean to make this change permanent?

Reply:
Oh!! my bad it was meant to be temporary.

Reply:
fixed with https://github.com/openmrs/openmrs-esm-patient-chart/commit/3b7377d5507f7d5e1a8e0fbc849ed4ff97d670cd